lib/block-supports/align.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Align block support flag.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Registers the attributes block attribute for block types that support it.
+ *
+ * @param  array $block_type Block Type.
+ */
+function gutenberg_register_alignment_support( $block_type ) {
+	if ( $block_type->supports && array_key_exists( 'align', $block_type->supports ) ) {
```

Comment:
I'm also wondering if there could be potential problems with this condition?  If someone were to define `"align": false` (although an odd case), this condition would not respect that declaration and assume the block has align support.

Reply:
Valid, I think we should make these tests more secure.

Reply:
Should be fixed.